### PR TITLE
Refactor 1

### PR DESCRIPTION
**Pull request by robbiegreiner**
These changes address a lot of repetitive code in my BeerCard, BreweryCard & CityCard components.  Instead of having huge conditionals with tons of repetitive code, I broke out the chunks into smaller conditionals.

### CODE REVIEW

**Comment by robbiegreiner** [comment]
Also addressed: I changed my string interpolation to ES6 format..

**Comment by bbp5280** [comment]
Nice clean up. Very readable.

**Comment by bbp5280** [comment]
Nice clean up. Very readable.

**Comment by robbiegreiner** [comment]
Addresses issue https://github.com/robbiegreiner/brewery-tracker/issues/24

**Review by lolakoala — APPROVED** [review]
Let's be real- I'm not looking through all 42 of these changed files. But it looks like more red than green, so if your app still works, more power to you!

File: src/components/BreweryCard.js [hunk 1]
```diff
@@ -3,45 +3,48 @@ import PropTypes from 'prop-types';
...
+  if (brewery) {
     return (
       <div className={ brewery.isFav ? 'beer-card brewery-card favorite-card' : 'beer-card brewery-card' }>
-        <Link to={'/brewery/' + brewery.id}>
+        <Link to={`/brewery/${brewery.id}`}>
           <h2 onClick={() => getBrewery(brewery.id)}>{brewery.name}</h2>
         </Link>
         <h3>{brewery.locations ? brewery.locations[0].locality + ", " + brewery.locations[0].region : ''}</h3>
-        <Link to={'/brewery/' + brewery.id}>
+        <Link to={`/brewery/${brewery.id}`}>
```
Comment (ameseee) [inline src/components/BreweryCard.js:30]: Yeah! Using that ES6!

File: src/components/CityCard.js [hunk 2]
```diff
@@ -52,7 +50,9 @@ CityCard.propTypes = {
   getBrewery: PropTypes.func,
   writeUserData: PropTypes.func,
   user: PropTypes.object,
-  removeFavoriteBrewery: PropTypes.func
```
Comment (ameseee) [inline src/components/CityCard.js:55]: using trailing commas will eliminate lines like these appearing as 'changed' 💯

File: src/components/Landing.js [hunk 3]
```diff
@@ -71,6 +71,9 @@ class Landing extends Component {
             featureBeer={this.props.featureBeer}
             setCurrentBeer ={this.props.setCurrentBeer}/>
         </div>
+        <footer>
+          <h3>© Robbie Greiner 2017</h3>
```
Comment (ameseee) [inline src/components/Landing.js:75]: DAYUM!

**Pull request merged**